### PR TITLE
Update rules.md

### PR DESCRIPTION
**Pull request by gruihuang**
Modify rules.md for bucket retention.

### CODE REVIEW

**Comment by joecheuk** [comment]
Hi @gruihuang , thanks for updating rules.md for the bucket retention scanner, do you mind also adding a brief description of the scanner to descriptions.md?

The full path to the file is:
_docs/_latest/configure/scanner/descriptions.md

**Comment by gruihuang** [comment]
Hi @joecheuk , I am working on the descriptions.md. It would be in several minutes.

File: _docs/_latest/configure/scanner/rules.md [hunk 1]
```diff
@@ -867,3 +867,49 @@ rules:
...
+    * **Valid values**: One of `organization`, `folder`, `project`
+      or `bucket`.
+
+  * `resource_ids`
+    * **Description**: A list of one or more resource ids to match.
+    * **Valid values**: String.
+
+* `minimum_retention`
+  * **Description**: The minimum number of days to remain data.
+    Remove this entry if it is not needed.
```
Comment (dufton) [inline _docs/_latest/configure/scanner/rules.md]: Mention that the rule must include a minimum, maximum or both.

File: _docs/_latest/configure/scanner/rules.md [hunk 2]
```diff
@@ -867,3 +867,52 @@ rules:
...
     want integers in this list.  Similarly, if the expression extracts
     a list of values, you need to provide lists.
+
+## Retention rules
+
+### Rule definition
+
+```yaml
+rules:
+  - name: retention policies should be in a specific range
```
Comment (umairidris) [inline _docs/_latest/configure/scanner/rules.md]: the rule should be more descriptive

maybe 'all buckets should have a retention policy for 100 to 200 days.

**Pull request merged**